dbt/adapters/dremio/api/handle.py

```diff
@@ -46,3 +41,8 @@ def close(self):
     def rollback(self):
         # todo
         logger.debug("Handle rollback not implemented.")
+
+    def __init__(self, parameters: Parameters):
```

Comment:
This is a constructor should should be before public methods according to our style guide, no?

Reply:
Oh whoops, got a little too excited with moving methods. I'll fix it, thanks.